Source/Project64/N64 System/Mips/Memory Virtual Mem.cpp

```diff
@@ -2281,59 +2276,54 @@ int CMipsMemoryVM::MemoryFilter( DWORD dwExptCode, void * lpExceptionPointer )
 	return EXCEPTION_EXECUTE_HANDLER;
 }
 
-int CMipsMemoryVM::LB_NonMemory ( DWORD PAddr, DWORD * Value, BOOL /*SignExtend*/ ) 
+bool CMipsMemoryVM::LB_NonMemory(DWORD PAddr, DWORD* Value, bool /*SignExtend*/)
 {
 	if (PAddr < 0x800000)
 	{
-		* Value = 0;
+		*Value = 0;
 		return true;
 	}
+
 	if (PAddr >= 0x10000000 && PAddr < 0x16000000) 
 	{
 		g_Notify->BreakPoint(__FILEW__,__LINE__);
 #ifdef tofix
 		if (WrittenToRom)
-		{

```

Comment:
I prefer to have the brackets, I know it adds more lines but I find it more consistent